tests/contrib/elasticsearch/test.py

```diff
@@ -117,11 +119,37 @@ def test_elasticsearch(self):
 
         assert len(result["hits"]["hits"]) == 2, result
 
+        # Raise error 404 with a non existent index
+        writer.pop()
+        try:
+            es.get(index=1000000, id=100)
```

Comment:
let's keep the test somewhat consistent with a real use-case. i.e. index is usually a string https://github.com/elastic/elasticsearch-py#example-use